lib/NotificationController.php

```diff
@@ -173,7 +173,12 @@ function() {
             'rplus_notifications_adapters_mandrill_apikey',
             __('Madrill API Key', 'rplusnotifications'),
             function() {
-                ?><input type="text" class="regular-text" id="rplus_notifications_adapters_mandrill_apikey" name="rplus_notifications_adapters_mandrill_apikey" value="<?php echo get_option( 'rplus_notifications_adapters_mandrill_apikey' ); ?>" /><?php
+            	if ( defined( 'RPLUS_NOTIFICATIONS_ADAPTER_MANDRILL_API_KEY' ) ) {
+            		_e( 'Mandrill API Key ist via wp-config.php gesetzt.', 'rplusnotifications' );
```

Comment:
Base language should be English

Reply:
Should be, aber ist es im ganzen Plugin nicht.